metricbeat/module/system/diskio/_meta/fields.yml

```diff
@@ -54,3 +54,58 @@
       type: long
       description: >
         The total number of of milliseconds spent doing I/Os.
+
+    - name: iostat.rrqmCps
```

Comment:
I'm divided here, as these names don't follow the [convention](https://www.elastic.co/guide/en/beats/libbeat/current/event-conventions.html), while they are pretty common for sysadmins. Would like to know @ruflin opinion on these.